src/cpptools.ts

```diff
@@ -201,37 +204,39 @@ export function parseCompileFlags(cptVersion: cpptools.Version, args: string[],
             const def = value.substring(2);
             extraDefinitions.push(def);
         } else if (value.startsWith('-std=') || lower.startsWith('-std:') || lower.startsWith('/std:')) {
```

Comment:
It looks like the check `else if (value.startsWith('-std=') || lower.startsWith('-std:') || lower.startsWith('/std:'))` doesn't need to happen unless `extractStdFlag` is true. It would be more efficient then to do:

```
else if (extractStdFlag && (value.startsWith('-std=') || lower.startsWith('-std:') || lower.startsWith('/std:')))
```